api/app/lib/json_web_token.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module JsonWebToken
+  def encode_token(payload)
+    JWT.encode(payload, ENV['JWT_SECRET'])
+  end
+
+  def decode_token(token)
+    JWT.decode(token, ENV['JWT_SECRET'], true, algorithm: 'HS256')
+  rescue JWT::DecodeError
+    nil
+  end
+end
```

Comment:
Is `/app/lib` a good place for this kind of module? Should this be refactored into a controller maybe? (not sure how that would look)

Reply:
This is fine so that you don't have to mess with configuration on things right now.  I generally put these in `lib`, you could argue this is a domain object and can put it in Models as well.  Anything you put in a nested folder in `app` will be autoloaded automatically under the app directory.